docs/known_issues.rst

```diff
@@ -75,6 +75,39 @@ Both will throw an exception if units do not cancel, e.g.::
 
 See: https://github.com/astropy/astropy/issues/7582
 
+Quantities Lose Their Units with Operations on pandas.Series
+------------------------------------------------------------
+
+Quantities may work with certain operations on `~pandas.Series` but
+this behaviour is not tested. Quantities are subclassed from ``numpy``'s `~numpy.ndarray`
+and they do not behave same as `~pandas.Series` objects.
+For example one know issue is that multiplying a `~pandas.Series` object
+with a unit will *not* return a ``Quantity`` object. It will return a `~pandas.Series`
+object without any unit::
+
+   >>> import pandas as pd # doctest: +SKIP
+   >>> import astropy.units as u # doctest: +SKIP
+   >>> a = pd.Series([1., 2., 3.]) # doctest: +SKIP
+   >>> a * u.m # doctest: +SKIP
+   0    1.0
+   1    2.0
+   2    3.0
+   dtype: float64
+
+One way around this is to use the in-place shift operator::
+
+   >>> a << u.m # doctest: +SKIP
+   <Quantity [1., 2., 3.] m>
+
+But this is fragile as this may stop working in future versions of
+pandas if they decide to override the dunder methods. What should be
```

Comment:
If we run doctests here with the development or latest version of `pandas` then we don't need to worry about this happening without us noticing.

Reply:
True, but I guess it's still nice to document the "right" way to initialize a quantity with a pandas object?

Reply:
I don't really know what to think of this sentence. By providing the example we are recommending using the shift operators. But the text is saying they are fragile and it's best not to use them. We shouldn't be making mixed statements like this.

Reply:
Hmm, @eerovaher, good point. I think it may still be OK to list the operator, but let's put `u.Quantity` first. I.e., something like,
```
To avoid this, best is just to initialize as a `Quantity`,

   >>> u.Quantity(a, u.m)
...
Note that the overrides pandas provides are not complete, and
as a consequence, using the (in-place) shift operator does work,
    >>> a << u.m
...
But this ...
```

Reply:
I like @mhvk suggestion. The reason I do think it makes sense to keep some docs about this fragile behavior is it seems like users are trying to use this today and they are surprised it works for some operations and it doesn't for some. The fragile behavior seems to stems from the user assumption that numpy array and pandas series should work similarly (?). 

Yes we can tell them to just use `u.Quantity(a, u.m)` directly but maybe it's nice to also explain why their expectation (Series ~ ndarray) doesn't work?